pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ublue_updater"
```

Comment:
Probably want to replace all instances of `ublue_updater` with `ublue_update` for consistency sake